.env

```diff
@@ -0,0 +1 @@
+PS_FF_PRODUCT_PAGE_V2=true
```

Comment:
Should be empty (maybe add a comment to explain why this file is added following the Symfony convention, but we have no default values to force for now)

Reply:
I'm more infavor of using a `.env.dist` file though, it follows a convention that is widely used in many other projects so I don't really understand why we should use `.env` as our dist file
Especially since `DotEnv` does handle the `.env.dist` file as its default fallback